### ISSUE TITLE
[Mechanic] Going through locked doors while dead

### ISSUE THREAD

**Issue by Dartheus** [dalaranwow/dalaran-wow#3961]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: You are able to bypass certain closed doors by getting killed next to them, then going through in ghost form and resurrecting

**Expected behaviour**: You should not be able to pass through doors that have the locked status when dead

**Steps to reproduce the problem**:

1.  Go to entrance to Karazhan without the key
2. Get killed by any of the mobs near the main entrance
3. Run through the gate and resurrect on the other side
4. A voila!  You've eliminated the need for the attunement quest

**Include proofs for this behaviour**

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
I believe this is unrelated to specific object ID

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
blizzlike behaviour http://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-exploits/12371-bypass-locked-doors.html

I'm not shure how this should be scripted. 

"This does NOT allow you to bypass instances that you must be 'attuned' for that happen to give a 'key' to allow attunement. Those will still not allow you to pass the instance barrier."

If you are in a group, and only one player has the key, the whole group is still able to enter Karazhan.

Set "On Hold" for now, more research is needed for the "how it should be" behaviour.
